Sources/ArcGISToolkit/Components/Offline/OfflineMapAreasView.swift

```diff
@@ -28,10 +28,19 @@ public struct OfflineMapAreasView: View {
     /// A Boolean value indicating whether the preplanned map areas are being reloaded.
     @State private var isReloadingPreplannedMapAreas = false
     
+    /// The currently selected map.
+    private var selectedMap: Binding<Map?>
+    
     /// Creates an `OfflineMapAreasView` with a given web map.
-    /// - Parameter map: The web map.
-    public init(map: Map) {
-        _mapViewModel = StateObject(wrappedValue: MapViewModel(map: map))
+    /// - Parameters:
+    ///   - onlineMap: The web map.
+    ///   - selectedMap: A binding to the currently selected map.
+    public init(
+        onlineMap: Map,
+        selectedMap: Binding<Map?>
```

Comment:
Remember not to repeat type information in parameter names. These should be `online` and `selection` (following SwiftUI's convention). Some with the corresponding properties.